### ISSUE TITLE
“-i,” the image directory command-line option, doesn’t find images

### ISSUE THREAD

**Issue by 0x00b1** [CellProfiler/CellProfiler#2684]
```sh
$ cellprofiler -r -c -p ExampleFlyImages/ExampleFly.cppipe -i ExampleFlyImages/images
Your pipeline version is 20140723174500 but you are running CellProfiler version 300. Loading this pipeline may fail or have unpredictable results.
CP-JAVA 15:19:08.824 [Thread-1] WARN  o.c.imageset.ChannelFilter - Empty image set list: no images passed the filtering criteria.
```

**Comment by 0x00b1** [comment]
The “-i” option updates the user’s “default input directory” preference.

I suspect that either the pathname isn’t valid, that setting isn’t modified, or CellProfiler doesn’t use that preference.

**Comment by mcquin** [comment]
Resolved #2716
